### ISSUE TITLE
A11y: link color on light theme

### ISSUE THREAD

**Issue by rudotriton** [yangshun/front-end-interview-handbook#261]
The yellow link color is slightly too light for the light theme and does not offer enough contrast against the white background to be considered accessible.

<img width="500" alt="light theme" src="https://user-images.githubusercontent.com/14181026/88159768-70151c80-cc16-11ea-92eb-aa0c28ee853d.png">

Dark theme with enough contrast for comparison:

<img width="500" alt="dark theme" src="https://user-images.githubusercontent.com/14181026/88160273-18c37c00-cc17-11ea-984e-cd5616b62923.png">

**Comment by shubhsk88** [comment]
I also had the same issue and would happy to correct this error

**Comment by thawsitt** [comment]
I also faced this accessibility issue and it is hard to read on the website in light theme. We should consider changing the theme color to increase the contrast ratio for better readability. 

The color contrast should be at least 4.5 according to [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast).

**Comment by yangshun** [comment]
Thank you for all the comments. I've changed the primary color to be pink on light theme and retained the current yellow only on dark theme.
